docs/learn/explanations.md

```diff
@@ -0,0 +1,8 @@
+# NeXus Definition Language (NXDL), XML, and YAML
+The NeXus data format, described by the NeXus Definition Language (NXDL), represents a concerted effort aimed at facilitating data exchange within scientific communities, particularly among those engaged in neutron, X-ray, and muon research [Könnecke, M. et al.](https://doi.org/10.1107/S1600576714027575). The data format is also being used by the material science community under the project [NeXus-FAIRmat proposal](https://fairmat-experimental.github.io/nexus-fairmat-proposal/) supporting FAIR (Findable, Accessible, Interoperable and Reuseable) data principles in material science. The format serves as a standardized framework for both data exchange and storage.
+
+At the NeXus data format's core, the NeXus Definition Language (NXDL) functions as the cornerstone through which scientists delineate the nomenclature and organizational structure of information within NeXus data files, tailored to specific scientific techniques. NXDL is used to define general data structures (base classes), which define the set of terms that might be used in an instance of that class. These base classes are the building blocks for defining, under the schema called application definition, measurement-specific or even instrument-specific or software-specific data storage objects. In this process, members and definitions of individual base classes can be used as is or customized. In essence, the process of schema development, whether for a base class or an application definition, entails crafting an NXDL schema definition file with the extension 'nxdl.xml', utilizing the Extensible Markup Language, [XML](https://www.w3.org/TR/REC-xml/REC-xml-20081126.xml).
```

Comment:
"The NeXus Definition Language (NXDL) at the core of the NeXus data format functions as ..."

Reply:
pieces of information

Reply:
personally would remove ", tailored to ..." cuz NXDL is always used irrespective if class was tailored or not